modules/core-updates/9.0.md

```diff
@@ -85,6 +85,38 @@ Some dependencies are still present but were upgraded which comes with their own
 - [Services translator, security.csrf.token_manager, session, session.flash_bag no longer public](https://github.com/PrestaShop/PrestaShop/pull/35069)
 - [Replace AdminSecurity annotation with attributes](https://github.com/PrestaShop/PrestaShop/pull/35212)
 
+### Kernel modifications
+
+Until PrestaShop 8.0 we relied on one Kernel that was used for the back office, in Symfony 9.0 we expanded the usage of the Symfony framework especially for two new features:
+- the new Admin API based on ApiPlatform and OAuth authentication
+- an experimental feature that gives access to a Symfony container on the frontend (in opposition to the custom built built container available today), this one is not very much used but it's a first stone for the future front migration
+
+Those two new environments have different mechanisms and configuration from the back office, to separate these configurations cleanly they each have their dedicated kernel class and their own configuration. This allows to cleanly define different routing, security configurations, dedicated services, listeners, ...
+Each kernel has a unique `applicationId` that allows to dynamize its configuration, its cache folder (since the services are not the same each kernel needs its own cache folder). The `appId` parameter has also been added globally to the `bin/console` tool. They all share the common `app/config/config.yml` configuration file but each one extends it in its own config folder.
```

Comment:
```suggestion
Each kernel has a unique `applicationId` that allows it to dynamize its configuration and cache folder (since the services are not the same, each kernel needs its own cache folder). The `appId` parameter has also been added globally to the `bin/console` tool. They all share the common `app/config/config.yml` configuration file, but each one extends it in its own config folder.
```